src/Antiforgery/src/Microsoft.AspNetCore.Antiforgery.csproj

```
@@ -7,7 +7,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;antiforgery</PackageTags>
     <IsPackable>false</IsPackable>
-    <Nullable>enable</Nullable>
+    <IsTrimmable>enable</IsTrimmable>
```

Trimming probably isn't enable for this project
```suggestion
    <IsTrimmable>true</IsTrimmable>
```